test/fs.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const metatests = require('metatests');
+const fs = require('fs');
+const { mkdirp } = require('..');
+const path = require('path');
+
+const testMkdirp = metatests.test('mkdir');
+const mkdirpTestDir = 'test/ex1/ex2';
+
+testMkdirp.beforeEach((test, cb) => {
+  fs.rmdir(mkdirpTestDir, err => {
+    if (err && err.code === 'ENOENT') {
```

Comment:
Should be
```suggestion
    if (err && err.code !== 'ENOENT') test.error(err);
```
just like in `testMkdirp.afterEach`.